ui/components/app/qr-hardware-popover/qr-hardware-popover.js

```diff
@@ -43,25 +39,7 @@ const QRHardwarePopover = () => {
   );
 
   const signRequestCancel = useCallback(() => {
-    let action = cancelTx;
-    switch (_txData.type) {
-      case MESSAGE_TYPE.PERSONAL_SIGN: {
-        action = cancelPersonalMsg;
-        break;
-      }
-      case MESSAGE_TYPE.ETH_SIGN: {
-        action = cancelMsg;
-        break;
-      }
-      case MESSAGE_TYPE.ETH_SIGN_TYPED_DATA: {
-        action = cancelTypedMsg;
-        break;
-      }
-      default: {
-        action = cancelTx;
-      }
-    }
-    dispatch(action(_txData));
```

Comment:
Do we not need to replace this with a `rejectPendingApproval`?

Reply:
Good catch, added that.